internal/tools/handler.go

```diff
@@ -2,20 +2,52 @@ package tools
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
+	"log"
 
 	"github.com/Azure/aks-mcp/internal/config"
 	"github.com/mark3labs/mcp-go/mcp"
 )
 
+// logToolCall logs the start of a tool call
+func logToolCall(toolName string, arguments interface{}) {
+	// Try to format as JSON for better readability
+	if jsonBytes, err := json.Marshal(arguments); err == nil {
+		log.Printf("\n>>> [%s] %s", toolName, string(jsonBytes))
+	} else {
+		log.Printf("\n>>> [%s] %v", toolName, arguments)
+	}
+}
+
+// logToolResult logs the result or error of a tool call
+func logToolResult(result string, err error) {
+	if err != nil {
+		log.Printf("    ERROR: %v", err)
+	} else if len(result) > 500 {
+		log.Printf("    Result: %d bytes (truncated): %.500s...", len(result), result)
```

Comment:
Could you also add tool name in the result log?

Reply:
done in [6e1898e](https://github.com/Azure/aks-mcp/pull/125/commits/6e1898ea3e56704508136f9ea6f11de10d458fb6)